integration/env/common.go

```diff
@@ -3,15 +3,16 @@ package env
 import (
 	"crypto/sha1"
 	"fmt"
-	"log"
 	"math/rand"
 	"os"
+	"path"
 	"regexp"
 	"strconv"
 	"strings"
 	"time"
 
-	"github.com/giantswarm/e2e-harness/pkg/framework"
+	"github.com/giantswarm/aws-operator/service"
+	v28 "github.com/giantswarm/aws-operator/service/controller/legacy/v28"
```

Comment:
Nit but should we use v30 since its the most recent copy?

Reply:
Please see https://github.com/giantswarm/aws-operator/pull/1997/files#diff-8be9c9110d12fbc697d102909cc5fc63R75-R85